src/app/shared/components/dialogs/twitch-edit/twitch-edit.component.html

```diff
@@ -42,7 +42,15 @@ <h2 mat-dialog-title> {{ data.id ? 'Update ' : 'Add ' }} a Twitch {{form.value.e
           </mat-form-field>
         </div>
 
-
+        <div class="example-container" *ngIf="form.value.event === twitchEvents[0]">
```

Comment:
if you'd use the enum, you could use it like  `twitchEvents.bits` instead of the array-way

Reply:
I tried to use the `twitchEvents.bits` enum value to check it but I just couldn't import it, always getting an error saying it is undefined

Reply:
yeah it needs to be added like the array `twitchEvents` just with the `ClassProp = Enum`  then its usuable .... unless the enum is `const` then you can just remove it to make it work